pkg/api/conversion.go

```diff
@@ -225,7 +222,8 @@ func Convert_resource_Quantity_To_resource_Quantity(in *resource.Quantity, out *
 }
 
 func Convert_map_to_unversioned_LabelSelector(in *map[string]string, out *unversioned.LabelSelector, s conversion.Scope) error {
-	if in == nil {
+	if *in == nil {
+		out = nil
```

Comment:
This is wrong - it should be `*out = nil`.  You're supposed to get passed a pointer to an object that always exists.